db/handle_duplicates.py

```diff
@@ -0,0 +1,47 @@
+from PIL import Image
+import hashlib
+from db import ImgGateway
+
+
+# Get unique hash from image
+def get_image_hash(path):
+    with Image.open(path) as img:
```

Comment:
also although this is cool to use image data in order to generate the hash, i think realistically we can just create a hash based on image name since the strings are guaranteed to be unique by selection anyways

Reply:
just a consideration i'm not strongly opinionated on this